certs/certs.c

```diff
@@ -28,7 +28,10 @@ const char certificates[] =
 "ksLi4xaNmjICq44Y3ekQEe5+NauQrz4wlHrQMz2nZQ/1/I6eYs9HRCwBXbsdtTLS\r\n"
 "R9I4LtD+gdwyah617jzV/OeBHRnDJELqYzmp\r\n"
 "-----END CERTIFICATE-----\r\n"
-/*DigiCert Global Root CA*/
+#ifndef LEAN_CERTS
+//Note: for devices with limited resources, other certs are defined out to save space. 
+//For Germany, China, and other regions, please change which certs are defined out to the appropriate ones.
```

Comment:
> and other regions [](start = 21, length = 18)

i don't believe there are other regions to consider

Reply:
Fixed.